snapcraft/tests/plugins/test_qmake.py

```diff
@@ -41,8 +41,8 @@ def setUp(self):
 
         class Options:
             options = []
-            qt_version = 'qt5'
             project_files = []
+            qt_version = 'qt5'
```

Comment:
@kalikiana oh, I'm sorry, I didn't noticed that (didn't move that line on purpose ;)). If you want to, I can revert that.